refresh/index.js

```diff
@@ -308,14 +308,16 @@ module.exports = Generator.extend({
       this.appInitCode = {
         capabilities: [],
         services: [],
+        service_imports: [],
+        service_variables: [],
```

Comment:
Not particularly important (could just be some tech debt for later), but the data layout/names here no longer really match the the usage.